deploy/buildtools/buildconfigfile.py

```diff
@@ -119,9 +120,6 @@ def __init__(self, args: Namespace) -> None:
 
     def setup(self) -> None:
         """Setup based on the types of build hosts."""
-        for build in self.builds_list:
-            auto_create_bucket(build.s3_bucketname)
-
         # check to see email notifications can be subscribed
         get_snsname_arn()
```

Comment:
I didn't move this. But shouldn't this be in `awsinit` (i.e. part of `managerinit`)?